game.js

```diff
@@ -0,0 +1,150 @@
+const readline = require('readline-sync')
+
+
+/** Game open, ask name
+ * 
+ * Let's play " Choices: ADITL as $(name)!"
+ * - take name assign as variable
+ * 
+ *"It's Wednesday, you get paid on Friday and you only have $13.75 to get you through the week! Let's make something happen ;-)" Are you ready? YN
+ * 
+ * -Y -> the time is 6:30 am. you have to be at work by 8:30 am and your commute is approximately and hour and seven minutes (1)
+ * 
+ * - N -> Thank you! Come Again!
+ * 
+ * 
+ * Hit snooze?
+ * -Y "Alright! You're more rested! The time is now 6:55 am"
+ * 
+ * 
+ */
+
+let userName = ""
+const promptUserForName = () => {
+    userName = readline.question("What are you called: ")
+    console.log('Hi ' + userName + ',')
+    console.log("")
+    console.log("It's Wednesday, you get paid on Friday and you only have $13.75 to get you through the week! Let's make something happen ;-)")
+    console.log("")
+}
+
+let cash = 13.75
+let time = 120
+let leaveHome = -22
+let transport = ""
+let userWillPay = ""
+
+
+
+const goWork = () => {
+
+    if (transport === 'train') {
+        if (userWillPay === 'yes') {
+            cash = cash - 2.75
+            console.log("You have $" + cash + " remaining")
+        } else if (userWillPay === 'no') {
+            console.log("Lucky!" + userName + ", you're on your way to work and saved money!")
+        }
+
+    } else if (transport === 'bus') {
+        if (userWillPay === 'yes') {
+            cash = cash - 2.75
+            console.log("You have $" + cash + " remaining")
+        } else if (userWillPay === 'no') {
+            console.log("Lucky!" + userName + ", you're on your way to work and saved money!")
+        }
+    }
+}
```

Comment:
This function ends your game. It should call a different function to continue to propel the game forward.